docker/docker_settings/test_mysql.py

```diff
@@ -0,0 +1,24 @@
+from .test_sqlite import *  # NOQA
```

Comment:
it is not obvious that `test_sqlite` file is base settings when you just look in file tree, I suggest moving base settings file to `base.py` and also do you think naming test settings files like `test_mysql.py` is redundant ?

Reply:
I was just following the conventions already in place. I like the idea of `base.py` 👍